temperature/models.py

```diff
@@ -0,0 +1,16 @@
+from sqlalchemy import Column, Integer, Float, func, ForeignKey, DateTime
+from sqlalchemy.orm import relationship
+
+from city.models import City
+from database import Base
+
+
+class Temperature(Base):
+    __tablename__ = "temperature"
+
+    id = Column(Integer, primary_key=True, index=True)
+    date_time = Column(DateTime, nullable=False, default=func.now())
+    temperature = Column(Float, nullable=False)
+    city_id = Column(Integer, ForeignKey("city.id"), nullable=False)
+
+    city = relationship(City, back_populates="temperatures")
```

Comment:
The `back_populates` attribute in the `city` relationship should match the attribute name in the `City` model that refers back to `Temperature`. Ensure that the `City` model has a corresponding `temperatures` attribute (e.g., `temperatures = relationship('Temperature', back_populates='city')`) to establish a bidirectional relationship.